Server/Bff.WebApi/Program.cs

```diff
@@ -62,74 +65,76 @@ private static WebApplicationBuilder SetupServices(string[] args)
-            app.UseCors(builder =>
-                        builder.AllowAnyOrigin()
-                               .AllowAnyHeader()
-                        );
-
-            app.UseSwagger();
-            app.UseSwaggerUI();
-            app.UseDeveloperExceptionPage();
-        }
-
-        private static void SetupWebApiDepencenyServices(WebApplicationBuilder builder, IKernel kernel)
-        {
-            builder.Services.AddSingleton(kernel);
-            builder.Services.AddSingleton(x => kernel.Get<IRovictLogger>());
-            builder.Services.AddSingleton(x => kernel.Get<IHandlerFactory>());
-            builder.Services.AddSingleton(x => kernel.Get<IOperationResultFactory>());
-            builder.Services.AddSingleton(x => kernel.Get<IExceptionHandler>());
-            builder.Services.AddSingleton(x => kernel.Get<IAdministrationContext>());
-        }
-
-        private static StandardKernel SetupDependecyInjection()
-        {
-            return new StandardKernel();
+            builder.Services.AddStartupTask<StartApplicationServerTask>();
+			
+			builder.Services.AddDbContext<AdministrationContext>();
+			
+            return builder;
+        }
+
+        private static void SetupProductionEnviroment(WebApplication app)
+        {
+            if (app.Environment.IsDevelopment())
+                return; 
+
+            // Setup Cors for some private adressen
+            app.UseCors(builder =>
+                        builder.AllowAnyOrigin()
+                               .AllowAnyHeader()
+                       );
+            app.UseHsts();
+        }
+
+        private static void SetupDevelopmentEnviroment(WebApplication app)
+        {
+            // Configure the HTTP request pipeline.
+            if (!app.Environment.IsDevelopment())
+                return;
+
+            // Disable Cors for development to make live simpler
+            app.UseCors(builder =>
+                        builder.AllowAnyOrigin()
+                               .AllowAnyHeader()
+                        );
+
+            app.UseSwagger();
+            app.UseSwaggerUI();
+            app.UseDeveloperExceptionPage();
+        }
+
+        private static void SetupWebApiDepencenyServices(WebApplicationBuilder builder, IKernel kernel)
+        {
+            builder.Services.AddSingleton(kernel);
+            builder.Services.AddSingleton(x => kernel.Get<IRovictLogger>());
+            builder.Services.AddSingleton(x => kernel.Get<IHandlerFactory>());
+            builder.Services.AddSingleton(x => kernel.Get<IOperationResultFactory>());
+            builder.Services.AddSingleton(x => kernel.Get<IExceptionHandler>());
+            builder.Services.AddSingleton(x => kernel.Get<IAdministrationContext>());
+        }
+
+        private static StandardKernel SetupDependecyInjection()
+        {
+            return new StandardKernel();
         }
+
+        public static string ServerUrl => "https://127.0.0.1:5101";
```

Comment:
Dit zou je in appsettings kunnen zetten en daaruit kunnen halen.

Reply:
Nope is tijdelijk